src/cdi/CDI.cc

```diff
@@ -885,7 +886,7 @@ CDI::SP_MultigroupOpacity CDI::mg(rtt_cdi::Model m, rtt_cdi::Reaction r) const {
  */
 CDI::SP_CPEloss CDI::eloss(rtt_cdi::CPModelAngleCutoff mAC, int32_t pz,
                            int32_t tz) const {
-  map_it entry = CPEloss_map.find(std::make_pair(pz, tz));
+  auto const entry = CPEloss_map.find(std::make_pair(pz, tz));
```

Comment:
Why is auto preferred here?

Reply:
The return type is fully defined by the type of `CPEloss_map`. Choosing the wrong type on the LHS will force a cast and possibly a deep object construction.